### ISSUE TITLE
gnomeconf2gdm can't change cursor theme

### ISSUE THREAD

**Issue by Anifyuli** [realmazharhussain/gdm-tools#12]
Hello, Bro Hussain.
I found issue again on `gnomeconf2gdm`. This feature can't change my cursor theme in gdm. I try to change `/usr/share/glib-2.0/schemas/99_manjaro-settings.gschema.override` and nothing happened. 

Cursor theme on GDM
![IMG_20220103_103059](https://user-images.githubusercontent.com/73734843/147898412-1b4e9452-07a5-4eb5-a36f-5f1f2529d097.jpg)

Cursor theme on desktop
![Screenshot from 2022-01-03 10-54-41](https://user-images.githubusercontent.com/73734843/147898626-2e202d80-7a9f-4e81-b26b-abe53aff4395.png)

If you have spare times, please fix that bug. Thanks for your hard works.

**Comment by realmazharhussain** [comment]
What files are present in your `/etc/dconf/db/gdm.d/` directory?

**Comment by Anifyuli** [comment]
List of files in `/etc/dconf/db/gdm.d/`

```
04-sound  
10-cursor-settings       
99-gnomeconf2gdm
05-power  
10_manjaro-gdm-branding
```

**Comment by realmazharhussain** [comment]
Can you send me your 99-gnomeconf2gdm file?

**Comment by Anifyuli** [comment]
Ok, Bro. Wait

```
#-------- Interface ---------
[org/gnome/desktop/interface]
#----------------------------
cursor-theme='volantes_light_cursors'
icon-theme='Colloid-dark'
show-battery-percentage=true
clock-show-weekday=false
clock-format='24h'

#-------- Sound ---------
[org/gnome/desktop/sound]
#------------------------
theme-name='freedesktop'
event-sounds=true
input-feedback-sounds=false
allow-volume-above-100-percent=false

#-------------- Touchpad ---------------
[org/gnome/desktop/peripherals/touchpad]
#---------------------------------------
speed=0.0
tap-to-click=true

#------------- Night Light --------------
[org/gnome/settings-daemon/plugins/color]
#----------------------------------------
night-light-enabled=true
night-light-temperature=uint32 3700
night-light-schedule-automatic=true
night-light-schedule-from=20.0
night-light-schedule-to=6.0

```

**Comment by realmazharhussain** [comment]
Did you install that cursor theme in your $HOME folder? If yes, then that is the problem.

If you do not know where that theme was installed, check `$HOME/.icons` and `$HOME/.local/share/icons` folders.

**Solution:** Move your theme from whereever it is installed to `/usr/share/icons`

**Comment by Anifyuli** [comment]
No, I install Volantes from AUR, Bro. And thats cursor theme located in `/usr/share/icons`

**Comment by realmazharhussain** [comment]
I need you to do following things

1. Tell me what package you use; `volantes-cursors`, or `volantes-cursors-git`?
2. Tell the output of the command `sudo ls -R ~gdm/.config/dconf`
   **Note:** it is `~gdm` not `~/gdm`
3. Create a zip file of your `/etc/dconf/db/gdm.d` directory and send it to me
   **Note:** If you want to send via email, send it to <realmazharhussain@gmail.com>

**Comment by realmazharhussain** [comment]
Sorry for being so late to reply. I only have limited data and I've been having problems with my data connection

**Comment by Anifyuli** [comment]
1. Bro @realmazharhussain, I install `volantes-cursors-git` from AUR

2. Output from `sudo ls -R ~gdm/.config/dconf` command
      ```
      /var/lib/gdm/.config/dconf:
      user
      ```

3. This files what you want to see it
    [gdm.d.zip](https://github.com/realmazharhussain/gdm-tools/files/7850129/gdm.d.zip)

4. For your reply, I feel thanksfull and no problem with everything. I'm sure I not angry with you, hahahaha

**Comment by realmazharhussain** [comment]
I can't figure out the problem. If I ever have any idea I will let you know.

By the way, Do other cursor themes have the same problem or just volantes?

**Comment by Anifyuli** [comment]
> I can't figure out the problem. If I ever have any idea, I will let you know.
> 
> By the way, Do other cursor themes have the same problem or just volantes?

All my cursor theme can't apply with gnomeconf2gdm. Sorry for late reply, my laptop has a problem recently

**Comment by realmazharhussain** [comment]
> Sorry for late reply

No need

**Comment by realmazharhussain** [comment]
Run these commands

```bash
gnomeconf2gdm
sudo rm /etc/dconf/db/gdm.d/10-cursor-settings
sudo dconf update
```

And let me know if it works

**Comment by realmazharhussain** [comment]
If you are still having this problem, you can try out the new [Login Manager Settings](https://realmazharhussain.github.io/gdm-settings) app.

And if that doesn't work either then may be it is a permission issue.
